source/guides/mosaic/creating-a-mosaic.rst

```diff
@@ -59,8 +57,7 @@ A) A :ref:`MosaicDefinitionTransaction <mosaic-definition-transaction>` to set t
         :start-after:  /* start block 01 */
         :end-before: /* end block 01 */
 
-B) A :ref:`MosaicSupplyChangeTransaction <mosaic-supply-change-transaction>`, to set the **initial supply**.
-We are going to create **1,000,000** mosaic units.
+2. Define a :ref:`MosaicSupplyChangeTransaction <mosaic-supply-change-transaction>` to set the **initial supply**. For instance, we are going to create **1,000,000** mosaic units.
```

Comment:
2. Define a :ref:`MosaicSupplyChangeTransaction <mosaic-supply-change-transaction>` to set the **initial supply**. For instance, we can set it to **1,000,000** mosaic units.